Tests/Packet++Test/PacketExamples/IPv4-encapsulated-IPv6.dat

```diff
@@ -0,0 +1 @@
+0100010000001a432000010008004500006e2e0100001029432d8b12192151834383600440e80032063f2001063809020001020102fffee275962002518343830000000000005183438300150402e537a964626bf4b35018816099d30000323030204e4f4f5020636f6d6d616e64207375636365737366756c2e0d0a
```

Comment:
Can you also include a `.pcap` file that contains this packet? It's easier to open it in Wireshark if needed...
Please note that [`.gitignore` disallows adding `.pcap` files](https://github.com/seladb/PcapPlusPlus/blob/master/.gitignore#L20-L24) so you need to use the `-f` flag in order to add it:

```shell
git add -f <YOUR_FILE>.pcap
```